docs/source/documentation/r_scripts.rst

```diff
@@ -0,0 +1,117 @@
+If you are not familiar with the Substra low-level library, you should read the
+:ref:`Substra introductory example <Titanic example>` first.
+
+.. caution:: This guide provides an easy to run some scripts in another language. The scripts are wrapped up in a Python process, so performances might be limited.
+
+Preparing the R script
+----------------------
+The inputs of your script are passed as arguments in the command line. This includes parameters (int, float or str) and
+(relative) file paths to data.
+
+The outputs of the scripts are written to stdout, and will be parsed later by the Python script.
+
+Below is an example of what your file should look like:
+
+.. code-block:: R
+    :caption: my_script.R
+
+    #!/usr/bin/env Rscript
+    args <- commandArgs()
+    # your script here
+    ...
+    write(outputs, "")
+
+
+Calling the R script from Python
+--------------------------------
+The Python script passed to Substra wraps the R script, so that it can be executed as a Python subprocess.
+The Python script reads the inputs defined as Substra ``FunctionInputSpec``, converts everything to string,
+appends all parameters in a command (``subprocess.run`` expects a list of str) and launched the subprocess.
+After the subprocess has finished, the output is cleaned.
+Everything printed to stdout in the R script is in the string ``raw_output.stdout``, depending on the type of output,
+additional cleaning steps might be required.
+Finally, the output is saved as a pickle file, to be shared with other nodes in the organisation.
+
+.. code-block:: Python
+    :caption: python_wrapper.py
+
+    import pickle
+    import subprocess
+    import substratools as tools
+
+
+    @tools.register
+    def run_script(inputs, outputs, task_properties):
+        data_file = inputs["data_file_path"]
+        param1 = str(inputs["param1"])
+        param2 = str(inputs["param2"])
+        raw_output = subprocess.run(['Rscript', 'my_script.R', data_file, param1, param2], capture_output=True)
+        model = int(raw_output.stdout.strip())
+        save_model(model, outputs["model"])
+
+
+    def save_model(model, path):
+        with open(path, "wb") as f:
+            pickle.dump(model, f)
+
+
+    if __name__ == "__main__":
+        tools.execute()
+
+Adapting the opener
+-------------------
+When using Substra with Python, the ``Opener`` object is used to load the data in memory.
+When using R, we don't need to load the data as Python objects in memory, so the opener simply returns the file path (or paths).
+
+.. code-block:: Python
+    :caption: opener.py
+
+    import pathlib
+    import substratools as tools
+
+    import os
+
+    class StubOpener(tools.Opener):
+        def fake_data(self, n_samples=None):
+            return ""
+
+        def get_data(self, folders):
+            return list(pathlib.Path(folders[0]).glob("*.csv"))
```

Comment:
Would it be clearer to update the import to `pathlib.Path` and here just use `Path(...)`?

Reply:
I'm of the team "explicit is better than implicit" when it comes to imported libraries, but I know that it is controversial because one can consider it affects negatively the readability. ⚔️